Public/Src/Cache/ContentStore/Interfaces/Logging/IOperationLogger.cs

```diff
@@ -92,8 +103,27 @@ public enum OperationKind
         /// <nodoc />
         public Exception Exception { get; }
 
+        /// <summary>
+        /// Tracing severity of the result.
+        /// </summary>
+        public Severity Severity { get; }

```

Comment:
Presuming this is for something in the future? I didn't see any usage of these two new fields.

Reply:
This will be used by CB.